content/r-package-dev/debugging.md

```diff
@@ -1,9 +1,8 @@
 ---
 author: David Watkins
-date: 9999-04-30
+date: 9999-10-15
 slug: debugging
 title: Debugging
-draft: True
```

Comment:
Yeah I guess.  It's in the .Rmd though, why does it change it?  Didn't we make a different template or something?